modules/3d/misc/python/warp_test.py

```diff
@@ -0,0 +1,113 @@
+import numpy as np
```

Comment:
I propose to migrate it to opencv_extra to have it together with test data itself. Python generators for dnn tests are placed there.